server/chat-commands/moderation.ts

```diff
@@ -678,8 +704,12 @@ exports.commands = {
 			message += `\n\nYour lock will expire in a few days.`;
 			targetUser.send(message);
 
-			const roomauth = Rooms.global.destroyPersonalRooms(userid);
-			if (roomauth.length) Monitor.log(`[CrisisMonitor] Locked user ${name} has public roomauth (${roomauth.join(', ')}), and should probably be demoted.`);
+			const roomauth = Rooms.global.destroyPersonalRooms(targetUser?.id);
```

Comment:
Why was this changed?

Reply:
It looks like you just didn't know how to type userid?

```suggestion
			const roomauth = Rooms.global.destroyPersonalRooms(userid);
```